ci/dash/build_src.sh

```diff
@@ -26,7 +26,8 @@ if [ "$CHECK_DOC" = 1 ]; then
     #test/lint/check-doc.py
     # Run all linters
     test/lint/lint-all.sh
-    test/lint/extended-lint-all.sh
+    # Don't run extended! It's way too slow! maybe later on dedicated runners
+    #test/lint/extended-lint-all.sh
```

Comment:
you should not disable it here in `ci/dash/build_src.sh` because it would affect also current gitlab runner.

Could you just add a command line argument instead, so, for github you can do not run them, for gitlab - run them.

Reply:
hmm.... true

Reply:
Actually; this isn't true. we should either drop the lint-extended-cppcheck (but then we have no extended linters and this script would break), or simply not run extended-lint-all.sh. I think this is the better solution for now.

Reply:
see: https://github.com/dashpay/dash/pull/6171/commits/4519c5ff40f9be4a756797dc272a58fd1c155b40 again